platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/camera/CameraAnimatorActivity.java

```diff
@@ -116,20 +138,82 @@ public void onAnimationUpdate(ValueAnimator animation) {
     return tiltAnimator;
   }
 
-  private static class LatLngEvaluator implements TypeEvaluator<LatLng> {
+  //
+  // Interpolator examples
+  //
 
-    private final LatLng latLng = new LatLng();
+  @Override
+  public boolean onCreateOptionsMenu(Menu menu) {
+    getMenuInflater().inflate(R.menu.menu_animator, menu);
+    return true;
+  }
 
-    @Override
-    public LatLng evaluate(float fraction, LatLng startValue, LatLng endValue) {
-      latLng.setLatitude(startValue.getLatitude()
-        + ((endValue.getLatitude() - startValue.getLatitude()) * fraction));
-      latLng.setLongitude(startValue.getLongitude()
-        + ((endValue.getLongitude() - startValue.getLongitude()) * fraction));
-      return latLng;
+  @Override
+  public boolean onOptionsItemSelected(MenuItem item) {
+    if (mapboxMap == null) {
+      return false;
+    }
+    findViewById(R.id.fab).setVisibility(View.GONE);
+
+    // reset camera to init position
```

Comment:
What about extracting this block of code into a `private` method and give it a name based on the comment? This way `onOptionsItemSelected` will be easier to read and understand and comment will become superfluous so it won't be necessary.